pkg/cloudprovider/cloudprovider.go

```diff
@@ -62,11 +62,11 @@ func NewCloudProvider(configFile string) (c *Client, e error) {
 		glog.Errorf("Get cloud env error: %+v", err)
 		return nil, err
 	}
-	oauthConfig, _ := adal.NewOAuthConfig(azureEnv.ActiveDirectoryEndpoint, azureConfig.TenantID)
+	oauthConfig, err := adal.NewOAuthConfig(azureEnv.ActiveDirectoryEndpoint, azureConfig.TenantID)
 	if err != nil {
-		return nil, fmt.Errorf("creating the OAuth config: %v", err)
+		glog.Errorf("creating the OAuth config: %+v", err)
```

Comment:
nit: Can we also change the error message to be `creating the OAuth config error` following the same pattern as rest of the error messages in that func?

Reply:
For sure–I updated it to `Create OAuth config error: %+v` to be totally parallel with the other local error messages.

Reply:
Thanks @mboersma